cmd/server/cmd/BUILD

```diff
@@ -8,6 +8,7 @@ go_library(
         "inventory.go",
         "root.go",
         "server.go",
+        "version.go",
```

Comment:
Can we add the exact same version command to mixc so we can tell whether it's compatible with mixs?